server/api.js

```diff
@@ -6,6 +6,43 @@ import { v4 as uuidv4 } from "uuid";
 const itemsPerPage = 25;
 const router = Router();
 
+const auth = async (req, res, next) => {
+	try {
+		const authHeader = req.headers.authorization;
+		const token = authHeader?.split(" ")[1];
+
+		if (!token) {
+			return res.status(401).json({ code: "AUTHRIZATION_REQUIRED" });
+		}
+
+		const sessionResult = await db.query(
+			"SELECT * FROM session WHERE token = $1",
+			[token]
+		);
+		const session = sessionResult.rows[0];
+
+		if (!session) {
+			return res.status(401).json({ code: "INVALID_SESSION" });
+		}
+
```

Comment:
You need to also check that the token hasn't expired

Reply:
Done